client/app/components/packages/landuse-form/edit.hbs

```diff
@@ -1,170 +1,178 @@
 <SaveableForm
-  @model={{@package.landuseForm}}
-  @validators={{array this.validations.SaveableLanduseFormValidations this.validations.SubmittableLanduseFormValidations}}
-  as |saveableForm|
+  @model={{@package}}
+  @validators={{array (hash) this.validations.SubmittablePackageFormValidations}}
```

Comment:
This shouldn't be reverted to (hash). `this.validations.SaveableLanduseFormValidations ` is useful and necessary

Reply:
Oops my mistake, wrong form